README.md

```diff
@@ -1,456 +1,492 @@
+[Coverage Status]: https://coveralls.io/repos/github/Microsoft/msphpsql/badge.svg?branch=dev
+[coveralls-site]: https://coveralls.io/github/Microsoft/msphpsql?branch=dev
+
+## Get Started
+
+* [**Ubuntu + SQL Server + PHP 7**](https://www.microsoft.com/en-us/sql-server/developer-get-started/php/ubuntu)
+* [**RedHat + SQL Server + PHP 7**](https://www.microsoft.com/en-us/sql-server/developer-get-started/php/rhel)
+* [**Windows + SQL Server + PHP 7**](https://www.microsoft.com/en-us/sql-server/developer-get-started/php/windows)
+* [**Docker**](https://hub.docker.com/r/lbosqmsft/mssql-php-msphpsql/)
+
+
+## Announcements
+
+ Please visit the [blog][blog] for more announcements.
+
+
+## Build (Windows)
+
+Note: if you prefer, you can use the pre-compiled binary found [HERE](https://github.com/Azure/msphpsql/releases)
+
+#### Prerequisites
+
+You must first be able to build PHP 7 without including these extensions.  For help with doing this, see the [official PHP website][phpbuild] for building your own PHP on Windows.
+
+#### Compile the drivers
+
+1. Copy the sqlsrv and/or pdo_sqlsrv source code directory from this repository into the ext subdirectory.
+
+2. Run `buildconf.bat` to rebuild the configure.js script to include the driver.
+
+3. Run `configure.bat --with-odbcver=0x0380 and the desired driver options (as below) [plus other options such as --disable-zts for the Non Thread Safe build]` to generate the makefile.  You can run `configure.bat --help` to see what other options are available.
+  * For SQLSRV use: `--enable-sqlsrv=shared`
+  * For PDO_SQLSRV use: `--enable-pdo=shared --with-pdo-sqlsrv=shared`
+
+4. Run `nmake`.  It is suggested that you run the entire build.  If you wish to do so, run `nmake clean` first.
+
+5. To install the resulting build, run `nmake install` or just copy php_sqlsrv.dll and/or php_pdo_sqlsrv.dll to your PHP extension directory.
+
+This software has been compiled and tested under PHP 7.0.20 and 7.1.6 using the Visual C++ 2015 compiler.
+
+## Install (Windows)
+
+#### Prerequisites
+
+- A Web server such as Internet Information Services (IIS) is required. Your Web server must be configured to run PHP
+- [Microsoft ODBC Driver 11][odbc11] or [Microsoft ODBC Driver 13][odbc13]
+
+#### Enable the drivers
+
+1. Make sure that the driver is in your PHP extension directory (you can simply copy it there if you did not use nmake install).
+
+2. Enable it within your PHP installation's php.ini: `extension=php_sqlsrv.dll` and/or `extension=php_pdo_sqlsrv.dll`.  If necessary, specify the extension directory using extension_dir, for example: `extension_dir = "C:\PHP\ext"`
+
+3. Restart the Web server.
+
+## Install (UNIX)
+The following instructions assume a clean environment and show how to install PHP 7.x, Microsoft ODBC driver, apache, and Microsoft PHP drivers on Ubuntu 15, 16, RedHat 7, Debian 8, and Mac OS X. 
+
+### Step 1: Install PHP7+ 
+
+#### PHP 7.0
+
+**Ubuntu 15.04, Ubuntu 15.10**
+

```

Comment:
I think our README is getting way too long, which somewhat defeats the purpose. Isn't it time to move all these instructions to [End to End instructions](https://github.com/Microsoft/msphpsql/wiki/Install-and-configuration#EndToEnd)?